wolfcrypt/src/port/arm/thumb2-sha256-asm_c.c

```diff
@@ -101,7 +101,7 @@ void Transform_Sha256_Len(wc_Sha256* sha256, const byte* data, word32 len)
         "STRD	r10, r11, [sp, #88]\n\t"
         /* Start of loop processing a block */
         "\n"
-    "L_SHA256_transform_len_begin:\n\t"
+    "L_SHA256_transform_len_begin%=:\n\t"
```

Comment:
Have you confirmed this works with compilers like IAR? Let me know if you need some help testing that.

Reply:
I haven't checked IAR and would really appreciate it if you could try it out!